src/assets/scripts/client/navigationLibrary/ProcedureModel.js

```diff
@@ -294,6 +294,38 @@ export default class ProcedureModel {
 
         return exitNames[randomIndex];
     }
+	
+	
+	  /**
+     * Return the name of the first entry point
+     *
+     * @for ProcedureModel
+     * @method getFirstEntryPoint
+     * @return {string}
+     */
+    getFirstEntryPoint() {
+        const entryNames = Object.keys(this._entryPoints);
```

Comment:
if you're grabbing keys off an object are expecting a certain sort order?  if this is coming off an object, _first_ entry is somewhat arbitrary because an objects keys are not going to be in a guaranteed order.

Reply:
If this is used for what I think it is, then _which_ element is returned is actually irrelevant. Looking into it. Would be in reference to the above comment:
> If the current expected departure runway is not valid for the requested SID, expect any runway which is valid. The aircraft must not be allowed to depart until it is taxiied to a runway which is valid for the SID.

Either way, there should definitely be some explanations to this effect in the docblock.

Reply:
assumption confirmed, docblock improved